packages/devtools_app/integration_test/test_infra/run/_in_file_args.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+
+enum InFileArgItems {
+  experimentsOn,
+  appPath,
+}
+
+/// Test arguments, defined inside the test file as a comment.
+class InFileArgs {
+  factory InFileArgs(String testFilePath) {
+    final content = File(testFilePath).readAsStringSync();
+    return InFileArgs.fromFileContent(content);
+  }
+
+  factory InFileArgs.fromFileContent(String fileContent) {
+    final values = _parseFileContent(fileContent);
```

Comment:
nit: name something other than values, as `values` is already a field name for the Map type. Reading `values.keys` below is a bit confusing at first